anki-study/README.md

```
@@ -0,0 +1,74 @@
+2. **처음부터 너무 많이 결정하지 않기** — 작게 시작 + trial and error로 점진 진화
+
+## 영감
+
+- <https://x.com/trq212/article/2052809885763747935>
+- <https://thariqs.github.io/html-effectiveness/>
+
+핵심: "LLM에게 HTML을 만들라고 시키면 효율적". 이걸 Anki 학습에 적용해보니 **준비 부담 0 = LLM에게 부탁만 하면 됨** 이라는 가치가 통했다 (첫 세션 평균 점수 63.6 / 100).
+
+## 디렉토리 구조
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)